roles/openshift_logging/tasks/main.yaml

```diff
@@ -28,6 +28,8 @@
   register: local_tmp
   changed_when: False
   check_mode: no
+  delegate_to: localhost
+  become: no
```

Comment:
`delegate_to` isn't doing anything here (and everywhere else we combine `local_action` and `delegate_to`), the big thing is `become: no/False`. We should remove the `delegate_to` and I'll remove them from the other `local_action` tasks we have.